src/verifiable/constants.ts

```diff
@@ -147,16 +147,15 @@ export enum SupportedPaymentProofType {
 /**
  * Media types for Payment supported currencies
  * @beta
+ * @deprecated
  * @enum {string}
  */
 export enum SupportedCurrencies {
```

Comment:
maybe make sense to remove it? I mean we are breaking PaymentRailsInfo, MultiChainPaymentConfig, PaymentRailsChainInfo